spec/Section 5 -- Validation.md

```diff
@@ -1494,7 +1500,7 @@ For example, the following query will not pass validation because `@skip` has
 been used twice for the same field:
 
 ```graphql counter-example
-query ($foo: Boolean = true, $bar: Boolean = false) {
+query($foo: Boolean = true, $bar: Boolean = false) {
```

Comment:
This is a prettier bug. Just filed https://github.com/prettier/prettier/issues/10655